src/platform/graphics/webgpu/webgpu-render-pipeline.js

```diff
@@ -72,10 +74,30 @@ const _stencilOps = [
 // temp array to avoid allocation
 const _bindGroupLayouts = [];
 
+/** @ignore */
+class CacheEntry {
+    /**
+     * Render pipeline
+     *
+     * @type {GPURenderPipeline}
+     * @private
```

Comment:
This doesn't look private to me. It's referenced from `WebgpuRenderPipeline`.

Reply:
it's private because it exposes GPURenderPipeline type and test:types fails.
I can either remove the type info, or make it private.

Reply:
Ah, makes sense... :)